src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -87,6 +89,32 @@ GraphingCalculator::GraphingCalculator()
     virtualKey->Key = (VirtualKey)187; // OemAdd key
     virtualKey->Modifiers = VirtualKeyModifiers::Control;
     ZoomInButton->KeyboardAccelerators->Append(virtualKey);
+    m_uiSettings = ref new UISettings();
+    m_uiSettings->ColorValuesChanged += ref new TypedEventHandler<UISettings ^, Object ^>(this, &GraphingCalculator::OnColorValuesChanged);
+
+    ApplicationDataContainer ^ localSettings = ApplicationData::Current->LocalSettings;
+    if (localSettings == nullptr)
+    {
+        return;
```

Comment:
Since this method is a hodge podge of different initialization steps, I don't think we should return early. Someone else might tack on more initialization to the bottom and not realize that the code may never run. Either wrap the logic in the negative of this if statement, or keep the existing logic and move all of this into it's own initialization method that gets called from here.

Reply:
I updated this to check if localSettings != nullptr.